src/pages/chainlink-nodes.astro

```diff
@@ -0,0 +1,98 @@
+---
```

Comment:
Is this change from .mdx to .astro for consistency reasons? I assume this only changes the extension and not the contents of the file.

Reply:
Yes, this change is made for consistency reasons.